lib/config.js

```diff
@@ -27,6 +27,20 @@ var config = convict(
         format: String,
         default: 'mem'
       }
+    },
+    oauth: {
+      host: {
+        format: String,
+        default: 'oauth.accounts.firefox.com'
+      },
+      port: {
+        format: 'port',
+        default: 443
+      },
+      insecure: {
+        format: Boolean,
+        default: false
```

Comment:
Probably worth throwing a comment on this one w.r.t to exactly what "insecure" entails :-)